warplib/src/warp_memory/bytes.cairo

```diff
@@ -0,0 +1,6 @@
+use warplib::warp_memory::WarpMemory;
+use warplib::warp_memory::WarpMemoryTrait;
+
+trait WarpMemoryBytesTrait{
+    fn bytes_to_fixed_bytes32(ref self: WarpMemory, location: felt252) -> u256;
```

Comment:
Shall we create our own `bytes32` type? Might help with readability and allow to implement specific traits for the type

Reply:
Yes, I think that is a good idea!

Reply:
More generally, can we preserve all solidity types?

Reply:
Yes, and with their own trait system which implements all solidity operations over them: like index for `bytesX`